test/Microsoft.AspNetCore.Mvc.FunctionalTests/Infrastructure/MvcTestFixture.cs

```diff
@@ -19,8 +21,15 @@ protected override void ConfigureWebHost(IWebHostBuilder builder)
                 .UseRequestCulture<TStartup>("en-GB", "en-US")
                 .UseEnvironment("Production")
                 .ConfigureServices(
-                    services => services.Configure<MvcCompatibilityOptions>(
-                        options => options.CompatibilityVersion = CompatibilityVersion.Version_2_1));
+                    services =>
+                    {
+                        var testSink = new TestSink();
```

Comment:
FYI users can now debug test execution with just two breakpoints: One after this line executes and one near the end of their test i.e. at least after `// Act`. Create an Object ID (a Watch doesn't help) here and see what `Writes` contains when the test completes.

(I also used this to make sure my change actually changed the loggers 😺)